parsers/thueringen.py

```diff
@@ -0,0 +1,103 @@
+from urllib.request import urlopen
+from bs4 import BeautifulSoup as parse
+import re
+from utils import Parser
+from datetime import datetime, timedelta
+from pyopenmensa.feed import LazyBuilder
+
+calendar_week_regex = re.compile('\d{4}-\d{1,2}$')
+week_start_end_date_regex = re.compile('\d{1,2}\.\d{1,2}\.\d{4}')
+fees_regex = re.compile('Bedienstete.*')
+amount_regex = re.compile('\d{1,2},\d{1,2}')
+days_regex = re.compile('day_\d$')
+ingredients_regex = re.compile('Inhalt:.*')
+
+
+def parse_url(url, today=False):
+	canteen = LazyBuilder()
+	content = urlopen(url).read()
+	document = parse(content, 'lxml')
+	employees_fee = 1.8
+	guests_fee = 3.2
+	fees = document.find_all('p', {'class': 'MsoNormal'})
+	for fee_candidate in fees:
+		fee_string = fees_regex.findall(fee_candidate.text)
+		for s in fee_string:
+			amount_strings = amount_regex.findall(s)
```

Comment:
This is deeply nested. Prefer extracting a function and name it after what happens here.

Reply:
I extracted and split the parsing into 3 different functions